package.json

```diff
@@ -40,6 +40,7 @@
     "emotion": "^10.0.27",
     "formik": "^2.1.4",
     "immer": "^7.0.9",
+    "lockr": "^0.8.5",
```

Comment:
Mmmm lockr is 4 years old (2 years without any maintanance) and it does not include typescript typings.

Should we remove this lib and use this TypeScript Gist?

https://gist.github.com/MichalZalecki/70a831304b0e83afbc0fbad3e8577068